Cargo.toml

```diff
@@ -23,15 +23,15 @@ repository = "https://github.com/software-mansion/scarb"
 anyhow = "1.0.71"
 assert_fs = "1.0.13"
 async-trait = "0.1.68"
-cairo-lang-compiler = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "0e81e72d" }
-cairo-lang-filesystem = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "0e81e72d" }
-cairo-lang-formatter = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "0e81e72d" }
-cairo-lang-language-server = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "0e81e72d" }
-cairo-lang-semantic = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "0e81e72d" }
-cairo-lang-sierra-to-casm = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "0e81e72d" }
-cairo-lang-starknet = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "0e81e72d" }
-cairo-lang-test-runner = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "0e81e72d" }
-cairo-lang-utils = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "0e81e72d" }
+cairo-lang-compiler = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "85866f6e" }
+cairo-lang-filesystem = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "85866f6e" }
+cairo-lang-formatter = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "85866f6e" }
+cairo-lang-language-server = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "85866f6e" }
+cairo-lang-semantic = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "85866f6e" }
+cairo-lang-sierra-to-casm = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "85866f6e" }
+cairo-lang-starknet = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "85866f6e" }
+cairo-lang-test-runner = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "85866f6e" }
+cairo-lang-utils = { version = "1.1.0", git = "https://github.com/starkware-libs/cairo.git", rev = "85866f6e", features = ["env_logger"]}
```

Comment:
You just want this commit https://github.com/starkware-libs/cairo/commit/7cbc6cf9d294b0e653085df296ba0330222b5f1c aren't you? If so, I would rather cherry-pick it to my branch with my backports: https://github.com/starkware-libs/cairo/compare/main...mkaput/1.1.0-test-runner-fix

Reply:
Yes! Cherry picking works for me if you prefer so

Reply:
OK, please use hash `43b83560`